salt/kubernetes-minion/kubeconfig.jinja

```diff
@@ -1,13 +1,12 @@
-{% set kube_servers = [] -%}
-{% for fqdn in salt['mine.get']('roles:kube-master', 'fqdn', expr_form='grain').values() -%}
-  {% do kube_servers.append(fqdn) -%}
-{% endfor -%}
+{% set api_server = "api." + salt['pillar.get']('internal_infra_domain','')  -%}
```

Comment:
I know, but I think there is no (easy/clean) way to just fail if the pillar value is empty...